src/unix/bsd/netbsdlike/netbsd/mod.rs

```diff
@@ -281,6 +285,419 @@ s! {
         pub ifm_index: ::c_ushort,
         pub ifm_data: if_data,
     }
+
+    // SCTP support
+    // sys/netinet/sctp.h
```

Comment:
without `sys/` here and in the next line

Reply:
You're right, I'll fix the comment -- thanks!